src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -632,4 +632,105 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
         return newCommands;
     }
+
+    winrt::Windows::Foundation::Collections::IVector<Model::Command> Command::ParsePowerShellMenuComplete(winrt::hstring json, int32_t replaceLength)
+    {
+        if (json.empty())
+        {
+            return nullptr;
+        }
+        auto data = winrt::to_string(json);
+
+        std::string errs;
+        static std::unique_ptr<Json::CharReader> reader{ Json::CharReaderBuilder::CharReaderBuilder().newCharReader() };

```

Comment:
Where do you keep copying this code from? 😅 
It's illegal, MSVC just happens to accept it!

```suggestion
        static std::unique_ptr<Json::CharReader> reader{ Json::CharReaderBuilder{}.newCharReader() };
```

I replaced all of the instances I could find in c4c046595e, but some more appeared overnight o_O

Reply:
(It's illegal because you are directly calling a constructor as a function)

Reply:
lmao you know this PR LONG predates that commit